### DISCUSSION TITLE
Kanboard Mobile App Bug

### DISCUSSION THREAD

**Discussion by Pray227** [kanboard/kanboard#5404] [General]
I tried to install the Kanboard Mobile application via the link https://apkcombo.com/kanboard-mobile/io.HarryCLucas.KanboardMobile/download/apk, but there was an error. Is there a fix for the apk yet?
Thank you

**Comment by fguillot** [comment]
There is no official Kanboard mobile app. Try to contact the author of this application.

**Reply by Pray227** [reply]
May I have the contact of the author of this application?

**Reply by rautamiekka** [reply]
No one knows, and googling was useless. Use the report button on that page.

**Reply by Pray227** [reply]
Finally I found Kanboard Mobile Apps via the following link https://khanos-a-kanboard-client.en.uptodown.com/android. In my opinion, this application is quite good, although it still needs to be refined with reporting or analytics features.
